dex/testing/zec/harness.sh

```diff
@@ -33,10 +33,10 @@ mkdir -p "${HARNESS_DIR}"
 
 WALLET_PASSWORD="abc"
 
-ALPHA_CLI_CFG="-rpcport=${ALPHA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass"
-BETA_CLI_CFG="-rpcport=${BETA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass"
-DELTA_CLI_CFG="-rpcport=${DELTA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass"
-GAMMA_CLI_CFG="-rpcport=${GAMMA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass"
+ALPHA_CLI_CFG="-rpcport=${ALPHA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass -conf=${ALPHA_DIR}/alpha.conf"
+BETA_CLI_CFG="-rpcport=${BETA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass -conf=${BETA_DIR}/beta.conf"
+DELTA_CLI_CFG="-rpcport=${DELTA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass -conf=${DELTA_DIR}/delta.conf"
+GAMMA_CLI_CFG="-rpcport=${GAMMA_RPC_PORT} -regtest=1 -rpcuser=user -rpcpassword=pass -conf=${GAMMA_DIR}/gamma.conf"
```

Comment:
What is missing from zcash-cli command that requires the `-conf`?  They work without this for me, although I'm still testing with zcashd 5.0.0 and 5.3.0 is out now...

Reply:
I'm on 5.2.0. On master I see this when I attempt to run the zec harness:
```
+ sleep 1
+ tmux send-keys -t zec-harness:4 './alpha z_importwallet /home/joe/git/dcrdex/dex/testing/zec/alphawallet ; tmux wait-for -S zec' 'C-m;' wait-for zec
+ tmux send-keys -t zec-harness:4 './beta z_importwallet /home/joe/git/dcrdex/dex/testing/zec/betawallet ; tmux wait-for -S zec' 'C-m;' wait-for zec
+ echo 'Generating 400 blocks for alpha'
Generating 400 blocks for alpha
+ tmux send-keys -t zec-harness:4 './alpha generate 400; tmux wait-for -S zec' 'C-m;' wait-for zec
+ ALPHA_ADDR=tmEgW8c44RQQfft9FHXnqGp8XEcQQSRcUXD
+ BETA_ADDR=tmSog4freWuq1aC13yf1996fy4qXPmv3GTB
++ ./delta getnewaddress
Error reading configuration file: Missing zcash.conf
+ DELTA_ADDR=
```

Reply:
Updating to 5.3.0 If thats out anyway.